package.json

```diff
@@ -65,7 +67,9 @@
     "tslib": "^1.10.0",
     "typescript": "^3.6.4"
   },
-  "dependencies": {},
+  "dependencies": {
+    "styled-components": "^4.4.1"
+  },
   "jest": {
```

Comment:
Can we make this a dev and peer dependency instead?

Reply:
I was wondering actually about this. Will this component library package assume that they will use styled components in the libraries that will use them?